staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -51,6 +51,7 @@ import (
 	policyv1 "k8s.io/api/policy/v1"
 	policyv1beta1 "k8s.io/api/policy/v1beta1"
 	rbacv1 "k8s.io/api/rbac/v1"
+	resourcev1beta2 "k8s.io/api/resource/v1beta2"
```

Comment:
The DRA controllers still use the `v1beta1` API, so we should probably use that here too. It is possible to use DRA without enabling the `v1beta2` API. We should update this when we update the controllers.

Reply:
We need to aim for `beta2` as it introduces the `spec.NodeName` as a server side filter. The DRA repo `master` branch is already using this version, so is just a matter of time for it to cut a version supporting `v1beta2` .
ON `v1beta1` we would have to do the filtering on the client side

Reply:
> We need to aim for beta2 as it introduces the spec.NodeName as a server side filter.

That's not true: all API versions support that filter.

Reply:
My bad, then I'll move back to v1beta1 for this PR

Reply:
Let's merge this with v1beta1 for consistency. We will then convert to v1 with the rest of the code during GA graduation.

Reply:
edited to v1beta1